src/wireframes/shapes/neutral/tabs.ts

```diff
@@ -130,26 +133,48 @@ export class Tabs implements ShapePlugin {
         p.setStrokeColor(ctx.shape);
     }
 
-    private parseText(ctx: RenderContext, fontFamily: string, fontSize: number, strokeThickness: number) {
-        const isRight = ctx.shape.getAppearance(TAB_ALIGNMENT) === TAB_ALIGNMENT_RIGHT;
-        const key = `${ctx.shape.text}_${fontFamily}_${fontSize}_${strokeThickness}_${isRight}`;
```

Comment:
Why is the caching dependent on that, is it because selected items are "longer"?